velox/connectors/hive/HiveConfig.h

```diff
@@ -35,10 +35,15 @@ class HiveConfig {
   static constexpr const char* kMaxPartitionsPerWriters =
       "max_partitions_per_writers";
 
+  /// Whether new data can be inserted into existing partitions.
```

Comment:
This comment is misleading. Velox doesn't currently allow appending data into existing partitions, only creating new partitions. This configuration property seems to allow appending data into unpartitioned tables only.

Reply:
Yes, you are right. I updated the comment.